packages/terra-compact-interactive-list/tests/jest/CompactInteractiveList.test.jsx

```diff
@@ -13,12 +15,235 @@ afterAll(() => {
   console.warn.mockRestore(); // eslint-disable-line no-console
 });
 
-describe('placeholder for tests', () => {
-  it('placeholder for a test', () => {
-    const wrapper = shallowWithIntl(
-      <CompactInteractiveList />,
-    ).dive();
+describe('Compact Interactive List', () => {
+  describe('fixed width list', () => {
+    const cols = [
+      {
+        id: 'Column-0',
+        displayName: 'Col_1',
+        width: 40,
+      },
+      {
+        id: 'Column-1',
+        displayName: 'Col_2',
+        width: 200,
+      },
+      {
+        id: 'Column-2',
+        displayName: 'Col_3',
+        width: 40,
+      },
+    ];
 
-    expect(wrapper).toMatchSnapshot();
```

Comment:
I thought we are still matching snapshots for the default component render once in our tests.

Reply:
I still need to add some tests, including an initial snapshot. Will get back to this comment once I have an update.